src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,57 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    private String login;
```

Comment:
Look carefully at the photo of the project, what changes are there, it's not critical, just be careful